### ISSUE TITLE
Min-Height of container is not calculated multiselect's tailwind styling

### ISSUE THREAD

**Issue by Enrica-r** [vueform/multiselect#118]
The multiselect-sinlge/muliple-label is positionned absolutly on top 0 and left 0. That's why the container doesn't adapt its size automatically to the content. As a work around Multiselect calculates the min-height in the class multiselect.

In the proposed Tailwind defintion there isn't such a calculation. 

My proposal is to avoid the absolut position and change the structure a little bit as described following:

Container class -> flex, justify content: space-between (instead of flex-end)
xxxLabel class -> flex-grow: 1

As consequence the clear and caret elements must be wrapped in a div tag. This div class must be a flexbox also. I have not checked the situation with tags. Probably it needs a inner container for all modes.

Thank you.

**Comment by adamberecz** [comment]
Could you elaborate? An example of that would be useful. You've noticed there's `spacer: 'h-9 py-px box-content'`, right? That supposed to guarantee the min height of the container.

**Comment by Enrica-r** [comment]
Thanks for your hint. I didn't see the spacer class. I entered the fixed height on the container class. Now I changed to the spacer element. But the kind of setting the heightness is very static.

All my other input elements has a padding around the content given by tailwind's standard base.css (input[text] = py-2 px-3) which produces a box of 42px height. OK, I could statically set your spacer to "h-10" (=40px + 2px border) equals 42px.

But if a end user changes the size of the font of his browser the tailwind input box will adapt dynamically the height of the box while the static spacer would nothing know about the font size. The be more dynamic is my intention of the proposal above.
